### PR TITLE
Update to Apache Server Configs v2.2.0

### PR DESCRIPTION
**Pull request by alrra**
:bowtie:

### CODE REVIEW

File: app/templates/htaccess [hunk 1]
```diff
@@ -258,14 +265,14 @@ AddDefaultCharset utf-8
 # | Clickjacking                                                               |
 # ------------------------------------------------------------------------------
 
-# Protect web site against clickjacking.
+# Protect website against clickjacking.
```
Comment (sindresorhus) [inline app/templates/htaccess:268]: important stuff

Comment (alrra) [inline app/templates/htaccess:268]: @sindresorhus believe it or not, someone complained about that :)

Comment (sindresorhus) [inline app/templates/htaccess:268]: When a project gets popular enough everything is under the microscope ;)

Comment (hemanth) [inline app/templates/htaccess:268]: :+1:

**Pull request merged**